planning/obstacle_avoidance_planner/README.md

```diff
@@ -776,6 +776,7 @@ $N_{circle}$ is the number of circles to check collision.
 - `option.is_showing_calculation_time` enables showing each calculation time for functions and total calculation time on the terminal.
 - `option.is_stopping_if_outside_drivable_area` enables stopping just before the generated trajectory point will be outside the drivable area.
 - `mpt.option.plan_from_ego` enables planning from the ego pose when the ego's velocity is zero.
+- `mpt.option.max_plan_from_ego_length` maximum length threshold to plan from ego. it is enabled when the length of trajecotry is shorter than this value.
```

Comment:
```suggestion
- `mpt.option.max_plan_from_ego_length` maximum length threshold to plan from ego. it is enabled when the length of trajectory is shorter than this value.
```

Reply:
oh, thanks I fixed it in https://github.com/autowarefoundation/autoware.universe/pull/1223/commits/1b5a8e124267eb7cc76d232567129f263341a9c0